ios/FluentUI/Vnext/Button/MSFButton.swift

```diff
@@ -11,6 +11,15 @@ import UIKit
 @objc open class MSFButton: ControlHostingView,
                             UIGestureRecognizerDelegate {
 
+    open override func didUpdateFocus(in context: UIFocusUpdateContext, with coordinator: UIFocusAnimationCoordinator) {
+        let style = state.style
+        guard style == .primary || style == .accentFloating, let stateImpl = state as? MSFButtonStateImpl else {
+            return
+        }
+
+        stateImpl.isFocused = !stateImpl.isFocused
```

Comment:
I originally tried `self == context.nextFocusedView` like with the other button, but `self` was never the next or previous focused view.

Reply:
I looked into this and that's because the view being focused is the first subview of self.

Here are some dumps of my debugger output:

**First event:**
------------------------------

(lldb) po context.previouslyFocusedView
▿ Optional<UIView>
  ▿ some : <_TtGC7SwiftUI14_UIHostingViewVS_7AnyView_: **0x61800004d880**; frame = (0 0; 60 60); gestureRecognizers = <NSArray: 0x604000218650>; layer = <CALayer: 0x6030001c96f0>>

(lldb) po context.nextFocusedView
▿ Optional<UIView>
  ▿ some : <_TtGC7SwiftUI14_UIHostingViewVS_7AnyView_: 0x618000044880; frame = (0 0; 93.6667 60.3333); focused = YES; gestureRecognizers = <NSArray: 0x604000241a50>; layer = <CALayer: 0x6030001a1580>>

(lldb) po self.subviews[0]
<_TtGC7SwiftUI14_UIHostingViewVS_7AnyView_: **0x61800004d880**; frame = (0 0; 60 60); gestureRecognizers = <NSArray: 0x604000218650>; layer = <CALayer: 0x6030001c96f0>>


**Second event...**
------------------------------

(lldb) po context.previouslyFocusedView
▿ Optional<UIView>
  ▿ some : <_TtGC7SwiftUI14_UIHostingViewVS_7AnyView_: 0x61800004d880; frame = (0 0; 60 60); gestureRecognizers = <NSArray: 0x604000218650>; layer = <CALayer: 0x6030001c96f0>>

(lldb) po context.nextFocusedView
▿ Optional<UIView>
  ▿ some : <_TtGC7SwiftUI14_UIHostingViewVS_7AnyView_: **0x618000044880**; frame = (0 0; 93.6667 60.3333); focused = YES; gestureRecognizers = <NSArray: 0x604000241a50>; layer = <CALayer: 0x6030001a1580>>

(lldb) po self.subviews[0]
<_TtGC7SwiftUI14_UIHostingViewVS_7AnyView_: **0x618000044880**; frame = (0 0; 93.6667 60.3333); focused = YES; gestureRecognizers = <NSArray: 0x604000241a50>; layer = <CALayer: 0x6030001a1580>>

Reply:
Looks like the focus is going to the hostingController's view, then. I've updated it to check if the *focusedView is a subview. Do we want it to check the hostingController's view directly, instead? We'd have to expose the hostingController as internal if we did.